forge-gui/res/cardsfolder/upcoming/triumph_of_saint_katherine.txt

```diff
@@ -0,0 +1,12 @@
+Name:Triumph of Saint Katherine
+ManaCost:4 W
+Types:Creature Human Warrior
+PT:5/5
+K:Lifelink
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ Praesidium Protectiva — When CARDNAME dies, exile it and the top six cards of your library in a face-down pile. If you do, shuffle that pile and put it back on top of your library.
+SVar:TrigExile:DB$ ChangeZone | Defined$ TriggeredNewCardLKICopy | Origin$ Graveyard | Destination$ Exile | RememberChanged$ True | SubAbility$ DBExileTopCard
```

Comment:
this can have `ExileFaceDown$ True` also... not as big a deal for digital, but might as well